src/main/java/net/sf/jabref/external/DroppedFileHandler.java

```diff
@@ -345,7 +345,7 @@ public void stateChanged(ChangeEvent arg0) {
 
         linkInPlace.setText(Localization.lang("Leave file in its current directory."));
         copyRadioButton.setText(Localization.lang("Copy file to file directory."));
-        moveRadioButton.setText(Localization.lang("Move file to file directory."));
+        moveRadioButton.setText(Localization.lang("Move file to file directory") + '.');
```

Comment:
The punctuation should be part of the language string as some languages have different symbols for the full stop (Chinese comes to my mind as an example).